helm/hello-world-app/Chart.yaml

```diff
@@ -1,8 +1,18 @@
 apiVersion: v2
 name: hello-world-app
-version: "0.1.0"
+version: "0.2.1"
+appVersion: "latest"
```

Comment:
What does `latest` mean? appVersion is optional so I would just leave this out.

Reply:
True, on the other hand we recommend setting it... I think I'll just take a longer route and do a proper release of the app first

Reply:
Yeah releasing this here should be better. Having latest seems like a worse example to me than having nothing.